autogen/main.tf.tmpl

```diff
@@ -208,9 +208,12 @@ resource "google_compute_backend_service" "default" {
     }
   }
 
-  log_config {
-    enable      = lookup(lookup(each.value, "log_config", {}), "enable", true)
-    sample_rate = lookup(lookup(each.value, "log_config", {}), "sample_rate", "1.0")
+  dynamic "log_config" {
+    for_each = lookup(lookup(each.value, "log_config", {}), "enable", false) ? [1] : []
```

Comment:
The former default is `enable = true` so we need to switch this back.

Reply:
done now, sir

Reply:
Also need to regenerate.

Reply:
apologies, I am a first timer in this repo and I forgot what was written in the contributing :/